src/mmap.cc

```diff
@@ -4,12 +4,15 @@
 #include "nan.h"
 #include "errno.h"
 
+#include <cinttypes>
+#include <cstdlib>
```

Comment:
Let's use `stdint.h` instead.

Reply:
@indutny Sure. Out of curiosity though, any specific reason why? Just preference?

\- Jonathan

Reply:
Less C++ features is better :wink: It also has shorter notation in the end.